playbook.yml

```diff
@@ -8,7 +8,7 @@
 - name: Set up AiiDA lab server
   hosts: all
   roles:
-    - role: marvel-nccr.aiidalab_server
+    - role: ansible-role-aiidalab-server
```

Comment:
One is the name of the git repo; the other is the name on ansible galaxy.
I'm pretty sure the old one is correct (correct me if I'm wrong)